db/db_iter.cc

```diff
@@ -343,17 +347,19 @@ class DBIter final: public Iterator {
   LocalStatistics local_stats_;
   PinnedIteratorsManager pinned_iters_mgr_;
   ReadCallback* read_callback_;
+#ifdef ROCKSDB_LITE
```

Comment:
why is this needed?

Reply:
Because this field is used just in LITE version.This will warn by compiler without UNUSED.